modules/images/regenerate-existing-images/load.php

```diff
@@ -7,3 +7,79 @@
  * @since n.e.x.t
  * @package performance-lab
  */
+
+/**
+ * Register the background job taxonomy.
+ *
+ * Intentionally on lower priority, so that other post types can be
+ * registered successfully and taxonomy can be used for all the post types.
+ *
+ * This taxonomy will be used as a queue system where each term represents a job in the queue.
+ *
+ * @return void
+ */
+function perflab_background_job_taxonomy() {
+	// Get all registered public post types.
+	$object_types = get_post_types(
+		array(
+			'public' => true,
+		),
+		'names'
+	);
+
+	// Labels for the background job taxonomy.
+	$labels = array(
+		'name'              => _x( 'Background Jobs', 'taxonomy general name', 'performance-lab' ),
+		'singular_name'     => _x( 'Background Job', 'taxonomy singular name', 'performance-lab' ),
+		'search_items'      => __( 'Search Background Jobs', 'performance-lab' ),
+		'all_items'         => __( 'All Background Jobs', 'performance-lab' ),
+		'parent_item'       => __( 'Parent Background Job', 'performance-lab' ),
+		'parent_item_colon' => __( 'Parent Background Job:', 'performance-lab' ),
+		'edit_item'         => __( 'Edit Background Job', 'performance-lab' ),
+		'update_item'       => __( 'Update Background Job', 'performance-lab' ),
+		'add_new_item'      => __( 'Add New Background Job', 'performance-lab' ),
+		'new_item_name'     => __( 'New Background Job Name', 'performance-lab' ),
+		'menu_name'         => __( 'Background Job', 'performance-lab' ),
```

Comment:
This one should be the plural, as the menu items should always be plural. Alternatively, we could just remove this entry as it should automatically be set to the value of `name` in that case.